Orange-Skeletons/welcome-to-nashville
# added api file in the scripts file

# Description

[Added api file on scripts]

Fixes # (issue)

just added a file in scripts (api-manager.js)

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout api-manager-v2`
4. [make sure the file (api-manager.js) exists in scripts folder]

# Checklist:
